### PR TITLE
Document import/export using ssh/sftp

### PR DESCRIPTION
**Pull request by benkay86**

### CODE REVIEW

File: docs/getting-started/import-export-data.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+title: Importing and Exporting Data
+created: 2023-09-15T09:29:00 (UTC -05:00)
+tags: []
+author: Benjamin Kay
+exclude: true
+---
+
+## Performance Considerations
+
+Importing and exporting data to/from the cluster is a necessary and routine activity for cluster users. However, please be aware that storing data locally on the cluster will always be faster than importing or exporting from another computer system, often by a considerable margin. Importing and exporting should ideally be an infrequent activity, with frequently-accessed data stored locally on the cluster for optimum performance. See [Storage Systems](import-export-data.md) for more information on cluster-local storage options and the [Accounting FAQ](faqs-accounting.md) for storage pricing options.
```
Comment (rcifadmin) [inline docs/getting-started/import-export-data.md:11]: fix: replace "import-export-data.md" with "storage-systems.md"

**Review by rcifadmin — CHANGES_REQUESTED** [review]
This looks great, thanks! I just noticed 1 hyperlink that needs to be changed

File: docs/getting-started/import-export-data.md (hunk 1 above)
Comment (benkay86) [reply docs/getting-started/import-export-data.md]: Thanks for reviewing!  Fixed hyperlink via force push, should be ready for merging now.

**Review by rcifadmin — APPROVED** [review]
LGTM

**Pull request merged**